### PR TITLE
HelloFresh case study

### PR DESCRIPTION
**Pull request by toan-hf**
To describe the case of HelloFresh

### CODE REVIEW

File: content/casestudy/hellofresh.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# HelloFresh logging platform serving multiple countries.
+
+TBD
```
Comment (repeatedly) [inline content/casestudy/hellofresh.md:3]: Do you have a plan to update this TBD?

Comment (toan-hf) [reply content/casestudy/hellofresh.md:3]: hi @repeatedly  i am working with my communication manager. she will give it a short summary in next few days. 
Thanks

Comment (toan-hf) [reply content/casestudy/hellofresh.md]: I updated

**Comment by repeatedly** [comment]
Thanks! Just merged!

**Pull request merged**